frontend/public/index.html

```diff
@@ -3,7 +3,33 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <title>Technigo React App</title>
+    <title>Secret Society of Arts and Craft</title>
+
+      <!-- Meta tags for facebook -->
+      <meta property="og:title" content="Neah and Linda's Project Authentication" />
+      <meta property="og:url" content="https://arts-and-craft-society.netlify.app" />
+      <meta property="og:image" content="https://arts-and-craft-society.netlify.app/images/ogimage.png" />
+      <meta property="og:image:width" content="1200" />
+      <meta property="og:image:height" content="630" />
+      <meta property="og:description" content="Join the Secret Society of Arts and Craft!" />
+      <meta property="og:locale" content="en" />
+  
+      <!-- Meta tags for twitter -->
+      <meta name="twitter:card" content="summary_large_image" />
+      <meta property="twitter:domain" content="arts-and-craft-society.netlify.app" />
+      <meta property="twitter:url" content="https://arts-and-craft-society.netlify.app" />
+      <meta name="twitter:title" content="Neah and Linda's Project Authentication" />
+      <meta name="twitter:description" content="Join the Secret Society of Arts and Craft!" />
+      <meta name="twitter:image" content="https://arts-and-craft-society.netlify.app/images/ogimage.png" />
+  
+      <!-- Favicon -->
+      <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon.png">
+      <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32.png">
+      <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16.png">
+      <link rel="manifest" href="/images/site.webmanifest">
+
```

Comment:
Nice that you made an effort and created these meta tags and browser window icon. It's the small things that makes a big difference.